app/forms/quota_search_form.rb

```diff
@@ -23,6 +23,18 @@ def page
     @page || 1
   end
 
+  def year
+    @year || DEFAULT_YEAR_VALUE
```

Comment:
We need to make sure this is dynamically worked out since a class constant is initialised when the application starts and we only deploy like once a week